### ISSUE TITLE
[bug]: compinit:527: no such file or directory: ~/.local/share/zinit/completions/_nvm

### ISSUE THREAD

**Issue by the-arkhive** [zdharma-continuum/zinit#647]
### What happened?

This error started printing twice each time I initiate a shell. The file specified does in fact exist.

### Steps to reproduce

Honestly I have no clue. This happened very randomly, but possibly after a recent oh-my-zsh update?

### Operating System & Version

OS: linux-gnu | Vendor: pc | Machine: x86_64 | CPU: x86_64 | Processor: unknown | Hardware: x86_64

### Zsh version

zsh 5.9 (x86_64-pc-linux-gnu)

### Terminal emulator

foot

### If using WSL on Windows, which version of WSL

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cands** [comment]
I experience the same issue and also thinks it stems from some recent update. I get one additional error printed:
`compinit:527: no such file or directory: ~/.local/share/zinit/completions/_nvm
compinit:shift:529: shift count must be <= $#`

**Comment by cands** [comment]
I investigated further and issuing command `zinit cclear` to clear completions list solved the issue for me.

**Comment by asgMostafa** [comment]
I get a similar error:
`tee: ~/.cache/zinit/completions/_docker: No such file or directory` 

`Running` `zinit cclear` didn't help also.

**Comment by voidbruxo** [comment]
I've succeed to somehow fix this problem with this workaround :
DIRECTORY="$HOME/.cache/zinit/completions/"
if [[ ! -d "$DIRECTORY" ]]; then
  # If it doesn't exist, create it
  mkdir -p "$DIRECTORY"
else
  clear
fi

**Comment by voidbruxo** [comment]
TL;DR : I had same error for this directory

**Comment by CanRau** [comment]
> I get a similar error: `tee: ~/.cache/zinit/completions/_docker: No such file or directory`
> 
> Running `zinit cclear` didn't help also.

same here, get this when I enable `zinit snippet OMZP::docker` in my `.zshrc`

**Comment by gzatko-brightpick-ai** [comment]
this fixed it:
```
mkdir  ~/.cache/zinit/completions
. ~/.zshrc
```

**Comment by zatkowich** [comment]
fixed:
```
mkdir  ~/.cache/zinit/completions
. ~/.zshrc
```
